src/Features/Core/Portable/EditAndContinue/EditSession.cs

```diff
@@ -762,7 +761,7 @@ public async ValueTask<SolutionUpdate> EmitSolutionUpdateAsync(Solution solution
                             pdbStream.ToImmutableArray(),
                             projectChanges.LineChanges,
                             updatedMethodTokens,
-                            updatedTypes: ImmutableArray<int>.Empty,
+                            updatedTypeTokens,
```

Comment:
I did this in main-vs-deps a while ago.. did this get lost in a merge?
https://github.com/dotnet/roslyn/blob/main-vs-deps/src/Features/Core/Portable/EditAndContinue/EditSession.cs#L764

Reply:
IDK :(